apps/nuxt3-ssr/components/VariableCard.vue

```diff
@@ -18,36 +18,28 @@ const resourcePathId = resourceIdPath(variableKey.value);
 
 <template>
   <article class="py-5 lg:px-12.5 p-5">
-    <header class="flex">
-      <div class="grow flex items-center">
-        <h2 class="min-w-[160px] mr-4 md:inline-block block">
+    <header class="flex md:flex-row gap-3 items-start md:items-center">
+      <div class="md:basis-1/5 p-2">
+        <h2 class="break-all">
           <NuxtLink
             :to="`/${schema}/ssr-catalogue/${catalogue}/variables/${resourcePathId}`"
             class="text-body-base font-extrabold text-blue-500 hover:underline hover:bg-blue-50"
           >
             {{ variable?.name }}
           </NuxtLink>
         </h2>
-
-        <span class="mr-4 text-body-base hidden md:block">
+      </div>
+      <div class="hidden md:flex md:basis-3/5">
+        <p class="text-body-base">
           {{ variable?.label }}
-        </span>
+        </p>
       </div>
-      <div class="flex">
```

Comment:
maybe keep the star ( add to favorite ) as a reverence for later

Reply:
Added it back in